libs/design-system/src/lib/Pages/LearnPage/index.tsx

```diff
@@ -0,0 +1,155 @@
+import { TutorialCardProps } from '../../Components/TutorialCard';
+import PageSections from '../shared/PageSections';
+import PageSection from '../shared/PageSection';
+import { useCallback, useMemo, useState } from 'react';
+import {
+  LargeVideoCard,
+  LargeVideoCardProps,
+} from '../../Components/VideoCard/LargeVideoCard';
+import {
+  SmallVideoCard,
+  SmallVideoCardProps,
+} from '../../Components/VideoCard/SmallVideoCard';
+import { ToggleButton } from '../../Components/ToggleButton';
+import { PaginatedTutorialCardList } from '../../Components/TutorialCard/PaginatedTutorialCardList';
+
+export type LearnPageProps = {
+  allTutorials: TutorialCardProps[];
+  cadenceHref: string;
+  cadenceTutorials: TutorialCardProps[];
+  nftTutorials: TutorialCardProps[];
+  videos: {
+    primary: LargeVideoCardProps;
+    secondary: SmallVideoCardProps[];
+  };
+};
+
+export function LearnPage({
+  allTutorials = [],
+  cadenceHref,
+  cadenceTutorials,
+  nftTutorials,
+  videos,
+}: LearnPageProps) {
+  const [filters, setFilters] = useState<string[]>([]);
+
+  const allTutorialsFiltered = useMemo(
+    () =>
+      filters.length > 0
+        ? allTutorials.filter(({ tags }) =>
+            filters.some((filter) => tags.includes(filter))
+          )
+        : allTutorials,
+    [allTutorials, filters]
+  );
+
+  const tags = Array.from(
+    new Set(allTutorials.reduce((acc, { tags }) => [...acc, ...tags], []))
+  ).sort();
+
+  return (
+    <div className="container w-full bg-primary-gray-50 p-6 dark:bg-black">
+      <PageSections>
+        <PageSection className="pt-0">
+          <LandingHeader
+            gradient="tools"
+            title="Learn"
+            buttonText="Button Text"
+            buttonUrl="#fixme"
+            callout="Featured"
+            description="Lorem ipsum dolor sit amet proin gravida lorem ipsum dolor sit."
+          />
+        </PageSection>
+
+        <PageSection className="flex-col items-stretch">
+          <div className="mb-6 flex items-baseline justify-between">
+            <h2 className="text-h2">Cadence</h2>
+            <ButtonLink
+              variant="secondary"
+              className="hidden whitespace-nowrap md:flex"
+              href={cadenceHref}
+              next
+              size="sm"
+            >
+              Go to Cadence
+            </ButtonLink>
+          </div>
+          <PaginatedTutorialCardList tutorials={cadenceTutorials} />
+          <ButtonLink
+            className="mt-6 w-full whitespace-nowrap md:hidden"
+            href={cadenceHref}
+            next
+            size="sm"
+          >
+            Go to Cadence
+          </ButtonLink>
+        </PageSection>
+
+        <PageSection className="flex-col items-stretch">
+          <h2 className="text-h2 f mb-6">NFTs</h2>
+          <PaginatedTutorialCardList tutorials={nftTutorials} />
+        </PageSection>
+
+        <PageSection>
+          <h2 className="text-h2 mb-6">Featured videos</h2>
```

Comment:
This needs a Go to YouTube button. I just added "external" and "right" icons to the button components that can be used here (https://github.com/onflow/next-docs-v1/pull/111/files#diff-74f71e61cdec7349c426ca754f9f12c3b91494f59e68f65fab35e2050a443964R16)

Reply:
Great catch. It looks like there's no "Go to Youtube" button on mobile, however. This mis probably an oversight since the other "header buttons" show up in mobile, so I'll make sure it's added there as well.